jjb/ui/edgex-ui-go.yaml

```diff
@@ -14,6 +14,8 @@
           go-root: '/opt/go-custom/go'
       - 'delhi':
           branch: 'delhi'
+      - 'edinburgh': 
+          branch: 'edinburgh'    
```

Comment:
It looks to me like this should have a pre_build_script, build_script, and go-root defined.

Reply:
Thanks @MightyNerdEric 
I have updated as requested.